recipes-examples/celebrity-face-match/demo-celebrity-face-match/demo-celebrity-face-match.service

```diff
@@ -6,6 +6,7 @@ After=systemd-udev-settle.service weston.service
 [Service]
 Type=simple
 User=root
+Environment="DISPLAY=:0"
```

Comment:
This should not be needed, as we are not using X11 but Wayland. Additionally, isn't `XDG_RUNTIME_DIR` required? Have you tested this?

Reply:
@NBendisch did the testing of the latest states and also that was installed to the fair demos last year etc. I would vote to anyway just merge it to kirkstone. It is anyways superseeded by your scarthgap release.

Reply:
Okay, agreed. We are not planning a kirkstone release, that's true.